payloadcms/payloadcms-prod/.env.example

```
@@ -0,0 +1,8 @@
+PROJECT=payloadcms-prod
+DATABASE_URI=mongodb://127.0.0.1/payloadcms-prod
+PAYLOAD_SECRET=<your-secret-here>
+
+PORT=3001
```

Can we use a different port so we can run the cms service at port 3000?

```suggestion
PORT=2338
```

Right now we have:

- The strapi admin for DEV will be available at http://localhost:1337/admin
- The strapi admin for PRO will be available at http://localhost:1338/admin
- The web app will be available at http://localhost:3000/
- The CMS service will be available at http://localhost:3001/